### PR TITLE
Feature/error

### PR DESCRIPTION
**Pull request by william-stacken**
Will override the default whitelabel error page with a custom one

### CODE REVIEW

File: src/test/java/se/kth/iv1201/group4/recruitment/presentation/error/ErrorHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    @Test
+    void testHttp404() throws Exception {
+        sendGetRequestWithStatusCode(mockMvc, "error", 404).andExpect(status().isNotFound())
+            .andExpect(isNotFoundErrorPage());
+
+    }
+
+    @Test
+    void testUnhandledHttpStatus() throws Exception {
+        sendGetRequestWithStatusCode(mockMvc, "error", 418).andExpect(status().isIAmATeapot())
```
Comment (fiilp) [inline src/test/java/se/kth/iv1201/group4/recruitment/presentation/error/ErrorHandlerTest.java:54]: 👍

Comment (william-stacken) [reply src/test/java/se/kth/iv1201/group4/recruitment/presentation/error/ErrorHandlerTest.java:54]: 🍵

File: src/main/resources/templates/error.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org"
+      th:with="lang=${#locale.language}" th:lang="${lang}">
+<head>
+  <meta charset="UTF-8">
+  <title>Error</title>
+</head>
+<body>
+  <main layout:fragment="content">
```
Comment (fiilp) [inline src/main/resources/templates/error.html:10]: `layout:fragment` is only needed if we want the error page to be decorated by a Thymeleaf layout .

**Review by fiilp — APPROVED** [review]
Not sure if there is a decoration missing on the error page. If not, then it looks good to go! 🚀🚀🚀

File: src/main/resources/templates/error.html (hunk 2 above)
Comment (william-stacken) [reply src/main/resources/templates/error.html:10]: Added decorations to the error page, opinion?

**Pull request merged**